test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -501,6 +513,26 @@ void BedrockPlugin_TestPlugin::upgradeDatabase(SQLite& db) {
     SASSERT(db.verifyTable("test", "CREATE TABLE test (id INTEGER NOT NULL PRIMARY KEY, value TEXT NOT NULL)", ignore));
 }
 
+void BedrockPlugin_TestPlugin::stateChanged(SQLite& db, SQLiteNodeState newState){
+    // We spin this up in another thread because `stateChanged` is called from the `sync` thread in bedrock
```

Comment:
There were a few typos that I fixed in my PR when I copied this comment https://github.com/Expensify/Auth/pull/8287/files#diff-f39431f92460cfbd78837114990de593b3991c91138ceb0426ba616061c4e2bcR550 you can probably just copy&paste that here